pipelines/data_embedding_pipeline/data_embedding_pipeline.py

```diff
@@ -12,9 +13,9 @@ def data_embedding_pipeline() -> None:
 
     Steps:
         fetch_data: A ZenML step which loads the data from a specified DVC data version.
-        embed_data: A ZenML step which embeds the text data into vectors and pushes to the vector database.
+        load_data: A ZenML step which embeds the text data into vectors and pushes to the vector database.
```

Comment:
I think there is a mistake here. Should `load_data` have the first description and `embed_data` have the second?